.vscode/settings.json

```diff
@@ -11,5 +11,8 @@
     "paket-files": true,
     "**/obj/**/en": true,
     "**/obj/**/pl": true
-  }
+  },
+  "[csharp]": {
```

Comment:
Not a fan of forcing a formatter (someone might want to use no formatter at all and format only before commit).